code/game/objects/items/devices/scanners.dm

```diff
@@ -968,14 +982,17 @@ GENE SCANNER
 	icon_state = initial(icon_state)
 	ready = TRUE
 
-/obj/item/sequence_scanner/proc/get_display_name(mutation)
+/obj/item/sequence_scanner/proc/get_display_name(mutation, active_detail=FALSE)
 	var/datum/mutation/HM = GET_INITIALIZED_MUTATION(mutation)
 	if(!HM)
 		return "ERROR"
 	if(discovered[mutation])
-		return  "[HM.name] ([HM.alias])"
+		return wrap_active_colour("[HM.name] ([HM.alias])", active_detail)
 	else
-		return HM.alias
+		return wrap_active_colour(HM.alias, active_detail)
+
+/obj/item/sequence_scanner/proc/wrap_active_colour(text, active_detail=FALSE)
+	return active_detail ? "<span class='green'>[text] - [active_detail]</span>" : text
```

Comment:
Wrap the span after calling the proc rather than inside of it. `<span>[wrap_colour()]</span>`